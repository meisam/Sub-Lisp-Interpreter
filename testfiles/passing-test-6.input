(CaR (QuOtE (a . B)))
(CONS 4 (QUOTE (A . B)))
