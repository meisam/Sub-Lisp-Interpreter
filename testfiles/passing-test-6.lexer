LispOpenParentheses
(
LispLiteralAtom
CAR
LispOpenParentheses
(
LispLiteralAtom
QUOTE
LispOpenParentheses
(
LispLiteralAtom
A
LispDot
.
LispLiteralAtom
B
LispCloseParentheses
)
LispCloseParentheses
)
LispCloseParentheses
)
LispOpenParentheses
(
LispLiteralAtom
CONS
LispNumericAtom
4
LispOpenParentheses
(
LispLiteralAtom
QUOTE
LispOpenParentheses
(
LispLiteralAtom
A
LispDot
.
LispLiteralAtom
B
LispCloseParentheses
)
LispCloseParentheses
)
LispCloseParentheses
)