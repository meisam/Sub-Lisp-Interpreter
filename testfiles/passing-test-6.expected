A
(4 . (A . B))
